src/utils/jwt.ts

```diff
@@ -35,19 +32,12 @@ export class JwtHelper {
 
   /**
    * Decodes JWT token and returns its payload.
-   * @param {string} encodedToken
-   * @return {?JsonObject|undefined}
    */
-  decode(encodedToken) {
+  decode(encodedToken: string): any {
```

Comment:
We should minimize use of `any`, especially in public APIs. If this is intended to be opaque, can you update the documentation to say so and change the return type to `{}`?

Reply:
Updated!

Reply:
In this case, I think `any` is appropriate since a JWT could have anything in its payload

Reply:
As a general utility, this needs to support a variety of JWTs. Perhaps the code calling this method could cast the value to a more specific type?

Reply:
Cool, I think `any` works well here. When we migrate JavaScript files that expects a certain kind of JSON in the payload, I think that could be a good time to define an `interface` for the expected JSON response. As for the no-any rule, maybe we could discuss that more offline?

Reply:
Updated to use `unknown` instead of `any`